README.md

```diff
@@ -359,6 +361,16 @@ curl -H 'Authorization: Bearer eyJhbGcxxxxx.eyJzdWIxxxxx.lWMJ66Wxx-xx' http://lo
 }
 ```
 
+Build image:
+```
+docker build -t suldlss/preservation_catalog:latest .
+```
+
+Publish:
+```
+docker push suldlss/preservation_catalog:latest
+```
+
```

Comment:
should we switch to circleci to do this automatically when PRs are merged to master?   Or use github actions?   If you think so, make a ticket (not critical path ... out of scope for this work cycle)

Reply:
Probably. I'll create a ticket.

Reply:
Boop: https://github.com/sul-dlss/preservation_catalog/issues/1406